GitCommands/PathUtil.cs

```diff
@@ -165,23 +155,9 @@ public static string NormalizePath(this string path)
         /// </remark>
         /// <param name="path">path to check.</param>
         /// <returns>path if it can be combined, empty otherwise.</returns>
-        public static string GetExtension(string? path)
+        public static string? GetExtension(string? path)
```

Comment:
Suggest removing this method completely

Reply:
PathUtil.Combine will be removed too

I wanted to have this approved and some other PRs merged before I made that change
So I will rebase and make the change and change this from draft (and then quickly merge after approval, assuming that opinions are raised on the draft...)